tests/unit/libbpf_test.cpp

```diff
@@ -2978,6 +2978,17 @@ TEST_CASE("BPF_PROG_BIND_MAP etc.", "[libbpf]")
     attr.prog_bind_map.flags = 0;
     REQUIRE(bpf(BPF_PROG_BIND_MAP, &attr, sizeof(attr)) == 0);
 
+    // Verify we can create a nested map.
+    attr.map_create = {};
+    attr.map_create.map_type = BPF_MAP_TYPE_ARRAY_OF_MAPS;
+    attr.map_create.key_size = 4;
```

Comment:
Did you mean sizeof(uint32_t) instead of 4?
Suggestion: To replace the key_size and value_size with sizeof(uint32_t).